siftseek/endpoints/seeker/apply/seeker_apply_get.py

```diff
@@ -0,0 +1,32 @@
+from flask import Response, jsonify
+
+from siftseek.endpoints.seeker import seeker
+from siftseek.endpoints.helpers.query_helpers import get_job_apps_by_seeker_id_or_404
+from siftseek.models.job_application import JobApplication
+from siftseek.schemas.job_application_schema import application_schema
+
+
+@seeker.get("/apply/<int:seeker_id>")
+def get_all_applications(seeker_id: int) -> tuple[Response, int]:
```

Comment:
i'd name this `get_all_applications_by_seeker_id`. Reading `get_all_applications` makes me think you're just retrieving ALL of the applications in the whole DB.

Reply:
Good catch, noted.